### PR TITLE
Send Apple Search Ad Timing DEVEX-486

### PR DESCRIPTION
**Pull request by E-B-Smith**

### CODE REVIEW

File: Branch-SDK/Branch-SDK/Branch.m [hunk 1]
```diff
@@ -977,55 +977,64 @@ - (BOOL)checkAppleSearchAdsAttribution {
...
-        if (attrDetails == nil) attrDetails = @{};
-        if (self.preferenceHelper.appleSearchAdDetails == nil)
-            self.preferenceHelper.appleSearchAdDetails = @{};
-        if (![self.preferenceHelper.appleSearchAdDetails isEqualToDictionary:attrDetails]) {
-            self.preferenceHelper.appleSearchAdDetails = attrDetails;
-            self.preferenceHelper.appleSearchAdNeedsSend = YES;
-        }
+            if (attrDetails.count > 0 && !error) {
+                [self.preferenceHelper addInstrumentationDictionaryKey:@"apple_search_ad"
+                    value:[[NSNumber numberWithInteger:elapsedSeconds*1000] stringValue]];
```
Comment (sojanpr) [inline Branch-SDK/Branch-SDK/Branch.m]: `elapsedSeconds`  is this in second or milli sec?

**Comment by sojanpr** [comment]
LGTM 👍

File: Branch-SDK/Branch-SDK/Branch.m (hunk 1 above)
Comment (E-B-Smith) [reply Branch-SDK/Branch-SDK/Branch.m]: It’s in seconds.

Comment (E-B-Smith) [reply Branch-SDK/Branch-SDK/Branch.m]: It’s in seconds.

**Comment by E-B-Smith** [comment]
Thanks!

**Pull request merged**